### PR TITLE
Update gsmp

### PR DESCRIPTION
**Pull request by huber-nicolas**

### CODE REVIEW

**Review by wurzka — APPROVED** [review]
Thanks, LGTM :)

File: file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
   "formatVersion": "1",
-    "header": {
+  "header": {
```
Comment (larshp) [inline file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json:3]: looks like the indentation is off?

Comment (huber-nicolas) [reply file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json:3]: To me it looks fine. 
Indendation is 2 spaces per level.
FormatVersion and header are the same level, so there should be no indentation

Comment (larshp) [reply file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json:3]: oh, okay

**Pull request merged**